src/Workspaces/Core/Portable/Workspace/Workspace_Editor.cs

```diff
@@ -591,7 +591,7 @@ protected internal void OnDocumentClosed(DocumentId documentId, TextLoader reloa
 
                         return oldSolution.WithDocumentTextLoader(documentId, data.reloader, PreservationMode.PreserveValue);
                     },
-                    data: (@this: this, documentId, reloader, updateActiveContext),
```

Comment:
flag not used (and is documented as such).  so no need to pass through to helper function.